src/common/Buttons.jsx

```diff
@@ -16,31 +16,40 @@ export const GoBack = () => {
 	const goBack = () => navigate(-1)
 
 	return (
-		<button className={btn.back} onClick={goBack}> <ImArrowLeft2 />back</button>
+		<button className={btn.back} onClick={goBack}> <ImArrowLeft2 />Back</button>
 	)
```

Comment:
Don`t just write your text like this, add it to a constants file.
 One more thing you could do is to include language support and save the inside en.json file (this is optional)